doc/listeners.rst

```diff
@@ -160,3 +160,29 @@ The following class can be used to read messages from BLF file:
 
 .. autoclass:: can.BLFReader
     :members:
+    
+  
+MF4 (Measurement Data Format v4)
+--------------------------------
+
+Implements support for MF4 (Measurement Data Format v4) which is a proprietary
+format from ASAM, widely used in many automotive software (Vector CANape, ETAS INCA, dSPACE COntrolDesk, etc.).
+
+MF4 support requires Python >= 3.6
+
+The data is stored in a compressed format which makes it compact.
+
+.. note:: Channels will be converted to integers.
+
+.. note:: MF4Writer does not suppport append mode
+
+`MF4Writer` has the following init arguments
```

Comment:
You could replace this section with

```suggestion
.. autoclass:: can.MF4Writer
    :members:
```

It adds the class documentation as well as the init parameters.

Reply:
Wouldn't that brake the builds for Python <3.6?

Reply:
Ah, good point. But the docs are built on Python 3.6, so it would work. Though we would have do clearly document that it is only available on Python 3.6+. (Which you have already done!)